src/apic-extension/azext_apic_extension/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.isPreview": true,
-    "azext.minCliCoreVersion": "2.51.0"
+    "azext.minCliCoreVersion": "2.56.0"
```

Comment:
I don't know why the min CLI core version was bumped here. Am I able to keep the min version unchanged? So the new version will not require users upgrade their Azure CLI.

Reply:
AAZ-dev tool might generate code that requires newly introduced functionalities in azure cli recent versions. 
If you reviewed the generated code and did not find any new features in aaz code, then you can keep the previous cli min version. Please run live tests and confirm its consistency.

Reply:
Thank you!